src/components/Users/Users.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <h1>Users</h1>
+    <div class="gender-filter">
+        <span>Filter by gender:</span>
+        <label><input type="radio" name="gender" v-model="filter.gender"> All</label>
+        <label><input type="radio" name="gender" value="male" v-model="filter.gender"> Male</label>
+        <label><input type="radio" name="gender" value="female" v-model="filter.gender"> Female</label>
+    </div>
+<!--    <v-radio-group :model="filter.gender">-->
```

Comment:
for some reason vuetify styles were messed up and I didn't spent too much time to make them work, sorry

Reply:
> for some reason vuetify styles were messed up and I didn't spent too much time to make them work, sorry

can u check with this config file?
```
import Vue from 'vue';
import Vuetify from 'vuetify';
import 'vuetify/dist/vuetify.min.css';

Vue.use(Vuetify);

export default new Vuetify({
  icons: {
    iconfont: 'mdi',
  },
});
```